docs/content/guides/developer/sui-101/using-events.mdx

```diff
@@ -154,6 +154,117 @@ A successful `curl` return
 
 The TypeScript SDK provides a wrapper for the `suix_queryEvents` method: [`client.queryEvents`](https://sdk.mystenlabs.com/typedoc/classes/_mysten_sui.client.SuiClient.html#queryEvents).
 
+<details>
+<summary>
+    TypeScript SDK queryEvents example
+</summary>
+
+```
+import {
+    useSignAndExecuteTransactionBlock,
+    useSuiClient,
+} from "@mysten/dapp-kit";
+import { TransactionBlock } from "@mysten/sui.js/transactions";
```

Comment:
This is the old version of the typescript SDk.  This should be updated to use `@mysten/sui`: https://sdk.mystenlabs.com/typescript/migrations/sui-1.0